src/modules/Core/selectors.js

```diff
@@ -115,6 +115,12 @@ export const buildExchangeRates = async (state: State) => {
       } else {
         finalExchangeRates[reverseExchangeRateKey] = 0
       }
+
+      // todo: remove when real rates available for FIO
+      if (key === 'FIO_iso:USD') {
```

Comment:
This will break for users not set to USD. This can be removed as we support a fake FIO rate in our core.

Reply:
yeah, I was added this before. removed.

also, when I'm changing default currency in settings it not changing fiat amount in wallet list, still showing in USD, is that ok?